### PR TITLE
9.0 cmis from 7.0

### PR DESCRIPTION
**Pull request by lmignon**
This module no longer depends on the connector module.
The goal is to have a base module to define the configuration for accessing a CMIS repository and providing a 'client' to communicate with this repository.
This module can thus be the basis for different integration scenarios with a CMIS container. (Via the connector module or via specialized fields etc)

### CODE REVIEW

File: cmis/__init__.py [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-from . import (
-    cmis_backend,
-    backend,
-    cmis_binding,
-)
+logger = logging.getLogger(__name__)
+
+logger.warning('Disable SSL Certificate Validation by python code')
+httplib2.Http = functools.partial(
+    httplib2.Http, disable_ssl_certificate_validation=True)
```
Comment (sbidoul) [inline cmis/__init__.py]: @lmignon is this global monkey patching necessary here?

File: cmis/models/cmis_backend.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        cmis fields that reference the backend.
+        """
+        ret = {}
+        for this in self:
+            ret[this.id] = self._get_web_description(this)
+        return ret
+
+    @api.multi
+    def get_cmis_repository(self):
+        """ Check the authentication with DMS """
```
Comment (sbidoul) [inline cmis/models/cmis_backend.py]: the doc string does not look right

File: cmis/__init__.py (hunk 1 above)
Comment (lmignon) [reply cmis/__init__.py]: not really. I'll remove it.

File: .travis.yml [hunk 3]
```diff
@@ -33,7 +36,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - pip install cmislib
+  - pip install git+https://github.com/lmignon/python-cmislib.git@6.0.dev#egg=cmislib
```
Comment (sbidoul) [inline .travis.yml]: @lmignon do we need your fork here?

**Comment by sbidoul** [comment]
Ok, that's the minimal foundation on which all cmis modules can build.

:+1:

**Comment by lmignon** [comment]
Ping @max3903. I'm looking for reviewers since this one is successfully used in production for 3 customers. :smirk:  @Olivier-LAURENT

**Review by Olivier-LAURENT — APPROVED** [review]
:+1: effectively @lmignon, deployed with success for an Alfresco backend. Thx.

**Pull request merged**